src/Shared/Recruit.Vacancies.Client/Infrastructure/Client/QaVacancyClient.cs

```diff
@@ -81,20 +81,29 @@ public async Task<QaDashboard> GetDashboardAsync()
             return dashboard;
         }
 
-        public async Task<List<VacancyReview>> GetSearchResultsAsync(string searchTerm)
+        public async Task<VacancyReview> GetSearchResultAsync(string searchTerm)
         {
             //The return type is a list to allow adding results from other searches to be implemented in future
             var result = new List<VacancyReview>();
 
-            if (!TryGetVacancyReference(searchTerm, out var vacancyReference)) return result;
+            if (TryGetVacancyReference(searchTerm, out var vacancyReference) == false) return null;
```

Comment:
can you return the result here which is an empty list. I don't like returning nulls, which means the consumer has to do more.